src/Bicep.LangServer/Snippets/SnippetsProvider.cs

```diff
@@ -175,7 +176,8 @@ private void CacheResourceDependencies(ImmutableHashSet<ResourceDependency> reso
             }
 
             string path = Path.GetFullPath(manifestResourceName);
-            SyntaxTree syntaxTree = SyntaxTree.Create(new Uri(path), template);
+            Uri uri = PathHelper.FilePathToFileUrl(path);
+            SyntaxTree syntaxTree = SyntaxTree.Create(uri, template);
```

Comment:
Discussed offline - since the 'manifestResourceName' is technically not a file system path or a URI, and also doesn't appear to be actually used in the logic here, if possible it feels like the safest cross-platform option would be to just stick a placeholder URI in here (something like `new Uri("inmemory://snippet.bicep")`)

Reply:
Updated here:
https://github.com/Azure/bicep/commit/dc6c35f044b6f4edb9699e8053711e14e5a4b9cd